### PR TITLE
Bug 1376506 - Make job row contents flush left like they used to be

### PR DESCRIPTION
**Pull request by kikisdeliveryservice**
**Fix for Treeherder Bug 1376506**
https://bugzilla.mozilla.org/show_bug.cgi?id=1376506
Bug Summary: the jobs section text was not left aligned. 

Please see link above for screenshots and more information.

### CODE REVIEW

File: ui/css/treeherder-job-buttons.css [hunk 1]
```diff
@@ -51,19 +52,25 @@
 }
 
 .group-content {
-  margin-left: -3px;
+  margin: 0 0;
+  white-space: nowrap;
+
```
Comment (tojon) [inline ui/css/treeherder-job-buttons.css]: This empty line shouldn't be here; the closing `}` of the class properties should be moved up one line.

File: ui/css/treeherder-job-buttons.css [hunk 2]
```diff
@@ -51,19 +52,25 @@
...
 
 .group-content::before {
   content: "(";
+  padding-left: 1px;
+
 }
 
 .group-content::after {
   content: ")";
+  margin-left: =-1;
```
Comment (tojon) [inline ui/css/treeherder-job-buttons.css]: Sorry I've seen negative margins but have never seen this before.

Is `=-1` intentional? I am curious what it means.

File: ui/css/treeherder-job-buttons.css [hunk 3]
```diff
@@ -3,7 +3,7 @@
  */
 
 .job-group {
-  margin: 0 -3px 0 3px;
+  margin: 0 0;
```
Comment (tojon) [inline ui/css/treeherder-job-buttons.css]: I think you could shorthand this to a single `0` is that is the intent. Or remove the margin property entirely if the job group element would inherit a margin of 0 from elsewhere.

File: ui/css/treeherder-job-buttons.css [hunk 4]
```diff
@@ -51,19 +52,25 @@
 }
 
 .group-content {
-  margin-left: -3px;
+  margin: 0 0;
```
Comment (tojon) [inline ui/css/treeherder-job-buttons.css]: Same here as on line 6 per the shorthanding.

**Comment by tojon** [comment]
If this helps also, you can run production in one tab, and your branch in the other, and do a blink test, flipping between the two with various job display settings.

For example, since your current changes touch job groups, turning on/off the global job group display in the navbar in both browser tabs, and blink testing the job tables in both states may be helpful. Or changing browser widths and row wraps.

Sometimes I also increase my browser zoom to absurdly large sizes so I can see in detail what is happening in the job rows.

I did a bit of testing with your branch and see some of the changes, I think there's perhaps still a bit of tweaking to do, per my drive-by comments. :) I defer to @camd though.

**Comment by kikisdeliveryservice** [comment]
Thank you for your comments @tojon I'll work on those and re-push!

**Comment by camd** [comment]
This looks good.  I think the main thing was taking the padding of the ``job-group`` but the other parts seem to help a bit, too.  Those pesky parentheses still wrap oddly.  They always have for us...  But This gets us into a better situation, so definitely helps.

**Comment by camd** [comment]
Thanks @tojon for the review help.  Saved me lots of time!  :)

**Comment by kikisdeliveryservice** [comment]
Thank you for your feedback @tojon and @camd !

**Pull request merged**